src/pydvl/value/oob/oob.py

```diff
@@ -0,0 +1,57 @@
+# Code written by @BastienZim
+# Bastien Zimmermann - bastien.zimmermann@craft.ai
```

Comment:
I think it's not good practice to put contributor names directly in the code. That puts additional pressure on other contributors and may prevent them from making to changes to that code. On top of that, there is already a [Github feature](https://github.com/aai-institute/pyDVL/graphs/contributors) that allows people to see the contributors and what contributions they made.